libsolidity/ast/Types.cpp

```diff
@@ -41,7 +41,7 @@
 #include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/algorithm/string/split.hpp>
-#include <boost/range/adaptor/reversed.hpp>
+#include <range/v3/view/reverse.hpp>
```

Comment:
Please put the `range` import after all boost imports. Best if it's sorted alphabetically.

Reply:
Done